CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 Change Log
 ==========
 
+Version 2.2.2 *(2020-09-24)* 
+----------------------------
+* We fixed a crash when starting `BeaconActivity` caused by a third-party library that we use internally – [Koin](https://github.com/InsertKoinIO/koin) – when using Kotlin version 1.4.x. We decided to use a beta version of  Koin to prevent the application from crashing. Please, let us know if you run into any issues. [#124](https://github.com/helpscout/beacon-android-sdk-sample/issues/124)
+* Users will no longer experience a crash when starting `BeaconActivity` when the `targetSdk` is `30`. [#132](https://github.com/helpscout/beacon-android-sdk-sample/issues/132)
```

Comment:
Is it worth calling out this too was due to 3rd party dependency Okhttp which had a issue detecting Android platform version?

Reply:
Good point. Will update.